tests/integration/test_integration_snapshots.py

```diff
@@ -248,10 +248,10 @@ def test_trace_with_wrong_meta_types_not_sent(meta):
     tracer = Tracer()
     with mock.patch("ddtrace.span.log") as log:
         with tracer.trace("root") as root:
-            root.meta = meta
+            root._meta = meta
```

Comment:
This test sets invalid tags and asserts and Exception is raised. Using Span.set_meta here will remove the invalid tags and cause the test to fail